src/components/ModularTable/ModularTable.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+// TODO:
+// import PropTypes from "prop-types";
+import { useTable } from "react-table";
+
+import Table from "../Table";
+import TableRow from "../TableRow";
+import TableHeader from "../TableHeader";
+import TableCell from "../TableCell";
+
+function ModularTable({ data, columns }) {
```

Comment:
Any suggestions for a better name are welcome.

This is meant to be a default "Table" component. `Table` (and `TableCell`, etc) is already taken by basic presentational components, `MainTable` is also taken as current "full featured" table.
It's going to be some time until "modular table" replaces all functionality of current `MainTable`, but I think at some point `MainTable` will become deprecated when new component provides enough features.

So, long story short - any suggestions for a name better than `ModularTable`?
`TableComponent`, `TableController`, `ReactTable` (this ties it a bit to internal react-table implementation, but this may be more confusing than helpful)...

Reply:
`ModularTable` sounds fine to me :) 

In the future when its to replace `MainTable` We can discuss if it can take over that namespace with a major version bump.

Reply:
+1 to `ModularTable`. I think we're all on the same page with what's meant by that.

Reply:
`ModularTable` it is then.